cfgov/unprocessed/css/organisms/mega-menu.less

```diff
@@ -599,6 +617,10 @@ body {
             }
         }
 
+        .hide-title {
+            visibility: hidden;
```

Comment:
That's a good point! I was thinking the additional titles would be useful to screenreaders since the content is in separate lists, but maybe they'd just be confusing. I added an `aria-hidden` attribute to the extra titles, which I think will hide them from screenreaders in most cases.